### PR TITLE
Release/v0.7.10

### PR DESCRIPTION
**Pull request by ytsssun**
**Issue number:**
n/a

**Description of changes:**
- Rebuild the control container image to pick up updates in Amazon Linux 2
- Pull the latest SSM Agent - upgrade to `3.3.131.0`.

**Testing done:**
- [x] lscpu detects number of socket: PASSED
- [x] Verify Bottlerocket release file is valid: PASSED
- [x] Enable admin container via script: PASSED
- [x] Bottlerocket package inventory is valid JSON: PASSED
- [x] Verify access to APIclient: PASSED
- [x] Bottlerocket package inventory is read-only: PASSED

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 0.7.10
+
+* Update SSM agent to 3.3.40.0 ([#58])
```
Comment (vyaghras) [inline CHANGELOG.md]: Update SSM agent to 3.3.131.0

Comment (ytsssun) [reply CHANGELOG.md]: Thanks, updated!

**Review by vyaghras — APPROVED** [review]

**Review by ginglis13 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 0.7.10
+
+* Update SSM agent to 3.3.131.0 ([#58])
+
+[#55]: https://github.com/bottlerocket-os/bottlerocket-control-container/pull/58
```
Comment (markusboehme) [inline CHANGELOG.md:5]: s/55/58/

Comment (ytsssun) [reply CHANGELOG.md:5]: Oops. Let me submit a new one for the update..

**Pull request merged**